### PR TITLE
Adding Astar, Kava, Tenet chains and fixing RPC for Klaytn

### PR DESCRIPTION
**Pull request by CJskii**
## PR-Codex overview
This PR adds support for three new chains: Kava, Tenet, and Astar. It also updates the Klaytn RPC URL.

### Detailed summary
- Added Kava chain
- Added Tenet chain
- Added Astar chain
- Updated Klaytn RPC URL

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by CJskii** [comment]
I have imported ``{defineChain}`` just like in any other network, but checks are failing for some reason. 

Could you please review it when you get a chance @jxom

File: src/chains/definitions/kava.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { defineChain } from '../../utils/chain.js'
```
Comment (jxom) [inline src/chains/definitions/kava.ts:1]: Please fix this import.

File: src/chains/definitions/kava.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export const kava = /*#__PURE__*/ defineChain({
+    id: 2222,
+    name: "Kava EVM",
+    network: "kava-mainnet",
+    iconUrl: "/chain-icons/kava.svg",
+    iconBackground: "#000000",
```
Comment (jxom) [inline src/chains/definitions/kava.ts:8]: These don't exist. Can you remove for other chains too?
```suggestion
```

Comment (CJskii) [reply src/chains/definitions/kava.ts]: This is removed now.

File: src/chains/definitions/kava.ts (hunk 1 above)
Comment (CJskii) [reply src/chains/definitions/kava.ts]: Updated to the latest location.

**Pull request merged**